spec/factories/sentence_details.rb

```diff
@@ -2,17 +2,23 @@
   factory :sentence_detail, class: 'Nomis::SentenceDetail' do
     initialize_with { Nomis::SentenceDetail.from_json(attributes.stringify_keys) }
 
-    paroleEligibilityDate { "2011-01-28" }
     releaseDate { "2011-01-28" }
     sentenceStartDate { "2011-01-28" }
-    tariffDate { "2011-01-28" }
     automaticReleaseDate { "2011-01-28" }
     postRecallReleaseDate { "2011-01-28" }
     conditionalReleaseDate { "2011-01-28" }
     homeDetentionCurfewEligibilityDate { "2011-01-28" }
     homeDetentionCurfewActualDate { "2011-01-28" }
     actualParoleDate { "2011-01-28" }
     licenceExpiryDate { "2011-01-28" }
+
+    trait :parole_eligibility_date do
+      paroleEligibilityDate { "2011-01-28" }
+    end
+
+    trait :tariff_date do
+      tariffDate { "2011-01-28" }
+    end
```

Comment:
I can see you've moved `paroleEligibilityDate` and `tariffDate` into traits, rather than being regular attributes on this factory. What was the reason for this? I'm sure it's correct – just curious!

Reply:
Hi @ollietreend I moved these two fields from being regular attributes to traits, because I realised when I created my new test, that checks to see if a parole eligible badge is being displayed if parole_review_date is present, this test passed before I had even created any test data. 
I then checked my two other tests that were also to do with displaying a parole eligible badge and realised that actually these tests were also both passing irrespective of how I set up my test data. In order for a parole eligible badge to display it needs either a parole_eligible_date, a tarriff_date, or a parole_review_date. So if just one of these is a regular attribute then all these tests will automatically pass. So it felt like having them as a regular attribute was undermining my tests. But I wasn't sure if making them a trait was the best practice or if their was a preferable way of doing it.

Reply:
@ollietreend I did worry when I was changing it if it was best practice!!